src/components/DateRangePicker.js

```diff
@@ -317,8 +317,7 @@ class DateRangePicker extends React.Component {
       <div className='mx-date-range-picker' style={styles.component}>
         <button
           aria-controls='calendarMenu'
-          aria-expanded={this.state.showSelectionPane}
-          aria-haspopup='menu'
```

Comment:
aria-haspopup='menu'  🤦‍♂

Reply:
is aria-haspopup='menu' valid? I see only boolean for it.

Reply:
`aria-haspopup` does allow the string `menu` in addition to `true` and `false`.  While `menu` is a valid value, it does have unexpected consequences in JAWS where it causes that screen reader to not call out a button as a button.

Also the standard does not make clear that you use the string `menu`. The only place that can it even explained is under the menu button section here https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html

In general I think we want to stick to just using `true` and `false` for `aria-haspopup` because of how JAWS deviates in it's interpretation of what it means.